docs/reference/kubo-rpc-cli.md

```diff
@@ -12,7 +12,7 @@ RPC API clients are available in multiple languages, and are listed below. You c
 
 | Language      | Client library                                            | Status              |
 |---------------|-----------------------------------------------------------|---------------------|
-| Go            | [ipfs/go-ipfs-api](https://github.com/ipfs/go-ipfs-api)                | Active              |
+| Go            | [ipfs/kubo/client/rpc](https://pkg.go.dev/github.com/ipfs/kubo/client/rpc)                | Active              |
```

Comment:
We want to discourage people from using `go-ipfs-api` and instead use and help improve `kubo/client/rpc`. Not sure if removing it totally is the best strategy, or if we should keep it as `Inactive`.

Reply:
@hacdias when we updated the docs to point users to Helia, we removed pointers to js-ipfs for the same reason. So just removing it is probably the best bet.